app/.classpath

```diff
@@ -45,14 +45,16 @@
 	<classpathentry kind="lib" path="lib/xml-apis-1.3.04.jar"/>
 	<classpathentry kind="lib" path="lib/xml-apis-ext-1.3.04.jar"/>
 	<classpathentry kind="lib" path="lib/xmlgraphics-commons-2.0.jar"/>
-	<classpathentry kind="lib" path="test-lib/junit-4.11.jar"/>
-	<classpathentry kind="lib" path="test-lib/fest-assert-1.2.jar"/>
-	<classpathentry kind="lib" path="test-lib/fest-reflect-1.2.jar"/>
-	<classpathentry kind="lib" path="test-lib/fest-swing-1.2.jar"/>
-	<classpathentry kind="lib" path="test-lib/fest-util-1.1.2.jar"/>
 	<classpathentry kind="lib" path="test-lib/jcip-annotations-1.0.jar"/>
 	<classpathentry kind="lib" path="lib/jtouchbar-1.0.0.jar"/>
 	<classpathentry kind="lib" path="lib/commons-lang3-3.8.1.jar"/>
 	<classpathentry kind="lib" path="lib/jssc-2.8.0-arduino4.jar"/>
+	<classpathentry kind="lib" path="test-lib/assertj-swing-junit-3.9.2.jar"/>
+	<classpathentry kind="lib" path="test-lib/assertj-core-1.7.1.jar"/>
+	<classpathentry kind="lib" path="test-lib/assertj-swing-1.2.0.jar"/>
+	<classpathentry kind="lib" path="test-lib/assertj-swing-junit-1.2.0.jar"/>
+	<classpathentry kind="lib" path="test-lib/assertj-swing-junit-4.5-1.2.0.jar"/>
+	<classpathentry kind="lib" path="test-lib/fest-util-1.2.5.jar"/>
+	<classpathentry kind="lib" path="test-lib/junit-4.5.jar"/>
```

Comment:
This seems to downgrade junit from 4.11 to 4.5. Is that intentional?

Reply:
It's a forced dependency from `assertj-swing-junit-4.5-1.2.0.jar`, unfortunately I didn't found an `assertj-swing-junit-4.11-1.2.0.jar`.

Maybe it's ok to build with `juint-4.11`, but since it's not a critical dependency I've just downgraded it.